tools/release/release_notes.py

```diff
@@ -0,0 +1,368 @@
+    file.write("---\n")
+    file.write("\n")
+    if langs_pr["nolang"]:
+        langs_pr["nolang"].sort()
+        file.write("\n".join(langs_pr["nolang"]))
+    else:
+        file.write("- None")
+    file.write("\n")
+    file.write("\n")
+    file.write(
+        "PRs going into release notes - please check title and fix in Github. Do not edit here.\n"
+    )
+    file.write("---\n")
+    file.write("\n")
+    if langs_pr["inrel"]:
+        langs_pr["inrel"].sort()
+        file.write("\n".join(langs_pr["inrel"]))
+    else:
+        file.write("- None")
+    file.write("\n")
+    file.write("\n")
+    file.write("PRs not going into release notes\n")
+    file.write("---\n")
+    file.write("\n")
+    if langs_pr["notinrel"]:
+        langs_pr["notinrel"].sort()
+        file.write("\n".join(langs_pr["notinrel"]))
+    else:
+        file.write("- None")
+    file.write("\n")
+    file.write("\n")
+
+
+def write_rel_notes(langs_pr, file, version, name):
+    file.write(rl_header.format(version=version, name=name))
+    if langs_pr["core"]:
+        file.write("Core\n---\n\n")
+        file.write("\n".join(langs_pr["core"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["c++"]:
+        file.write("C++\n---\n\n")
+        file.write("\n".join(langs_pr["c++"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["c#"]:
+        file.write("C#\n---\n\n")
+        file.write("\n".join(langs_pr["c#"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["go"]:
+        file.write("Go\n---\n\n")
+        file.write("\n".join(langs_pr["go"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["Java"]:
+        file.write("Java\n---\n\n")
+        file.write("\n".join(langs_pr["Java"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["node"]:
+        file.write("Node\n---\n\n")
+        file.write("\n".join(langs_pr["node"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["objc"]:
+        file.write("Objective-C\n---\n\n")
+        file.write("\n".join(langs_pr["objc"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["php"]:
+        file.write("PHP\n---\n\n")
+        file.write("\n".join(langs_pr["php"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["python"]:
+        file.write("Python\n---\n\n")
+        file.write("\n".join(langs_pr["python"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["ruby"]:
+        file.write("Ruby\n---\n\n")
+        file.write("\n".join(langs_pr["ruby"]))
+        file.write("\n")
+        file.write("\n")
+    if langs_pr["other"]:
+        file.write("Other\n---\n\n")
+        file.write("\n".join(langs_pr["other"]))
+        file.write("\n")
+        file.write("\n")
+
+
+def build_args_parser():
+    import argparse
+    parser = argparse.ArgumentParser()
```

Comment:
Write a small description here might be helpful. Personally, I would like to know "rel" means "release" when I execute this script.

Reply:
Done

Reply:
Changed rel_ to release_. The help string has the description.